src/test/ui/issue-23302-2.stderr

```diff
@@ -1,20 +1,11 @@
-error[E0391]: cyclic dependency detected
-  --> $DIR/issue-23302-2.rs:14:9
-   |
-LL |     A = Y::B as isize, //~ ERROR E0391
-   |         ^^^^^^^^^^^^^ cyclic reference
-   |
-note: the cycle begins when const-evaluating `Y::A::{{initializer}}`...
-  --> $DIR/issue-23302-2.rs:14:9
-   |
-LL |     A = Y::B as isize, //~ ERROR E0391
-   |         ^^^^^^^^^^^^^
-note: ...which then requires computing layout of `Y`...
+error[E0391]: cycle detected when const-evaluating `Y::A::{{initializer}}`
   --> $DIR/issue-23302-2.rs:14:9
    |
 LL |     A = Y::B as isize, //~ ERROR E0391
    |         ^^^^
-   = note: ...which then again requires const-evaluating `Y::A::{{initializer}}`, completing the cycle.
+   |
+note: ...which requires computing layout of `Y`...
```

Comment:
I believe this note should have a span pointing at [line 13](https://github.com/Zoxc/rust/blob/ee64ecafe51f99e6f243ae4fee3241ffd00b914c/src/test/ui/issue-23302-2.rs#L13).